test_scripts/unit_tests/utils.bats

```diff
@@ -0,0 +1,192 @@
+setup() {
+    load '/usr/local/lib/bats-support/load.bash'
+    load '/usr/local/lib/bats-assert/load.bash'
+
+    DIR="$( cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd)"
```

Comment:
FYI, this is usually "good enough" but not bullet proof.

Reply:
That's from the BATS documentation, so i'm open to suggestions here.

Reply:
Yeah a lot of people use it, just it can go wrong once you get into symlinks and mount points... but not relevant here.